swap_meet/vendor.py

```diff
@@ -1,2 +1,78 @@
+from swap_meet.item import Item
+from operator import attrgetter
```

Comment:
Nice job researching and implementing a new library that we haven't gone through in class!